docs/resources/cluster.md

```diff
@@ -44,7 +49,18 @@ Enables the provisioning and management of Redpanda clusters on AWS and GCP. A c
 
 Required:
 
-- `allowed_principals` (List of String)
+- `allowed_principals` (List of String) The ARNs of the allowed principals
+- `connect_console` (Boolean)
```

Comment:
What does connect_console do?

Reply:
Updated the description from the api docs